docs/platforms/python/crons/index.mdx

```diff
@@ -12,6 +12,70 @@ Sentry Crons allows you to monitor the uptime and performance of any scheduled,
 
 <PlatformContent includePath="crons/setup" />
 
+## Check-Ins
+
+Check-in monitoring allows you to track a job's progress by capturing two check-ins: one at the start of your job and another at the end of your job. This two-step process allows Sentry to notify you if your job didn't start when expected (missed) or if it exceeded its maximum runtime (failed).
+
+If you use the `monitor` decorator/context manager, the SDK will create check-ins for the wrapped code automatically.
+
+```python
+from sentry_sdk.crons import capture_checkin
+from sentry_sdk.crons.consts import MonitorStatus
+
+check_in_id = capture_checkin(
+    monitor_slug='<monitor-slug>',
+    status=MonitorStatus.IN_PROGRESS,
+)
+
+# Execute your task here...
+
+capture_checkin(
+    monitor_slug='<monitor-slug>',
+    check_in_id=check_in_id,
+    status=MonitorStatus.OK,
+)
+```
```

Comment:
Not sure why this section is here. Maybe call it "Manual check-ins" and move it below the "Upserting Cron Monitors"